pkg/handlers/update.go

```diff
@@ -217,6 +217,22 @@ func (h *Handler) UpdateAdminConsole(w http.ResponseWriter, r *http.Request) {
 		Success: false,
 	}
 
+	isKurl, err := kotsadm.IsKurl()
+	if err != nil {
+		logger.Error(errors.Wrap(err, "failed to check kURL"))
+		updateAdminConsoleResponse.Error = err.Error()
```

Comment:
let's not return the error stack trace to the user? how about a hardcoded message?
```suggestion
		updateAdminConsoleResponse.Error = "failed to check kURL"
```

Reply:
actually, why is there another function that doesn't return any errors? https://github.com/replicatedhq/kots/pull/2590/files?w=1#diff-eaeff8d34bc12281033f61dfba86f59d58aa83762ef0af09fb275bd5347fd8bfR11